angular/wolox-books/src/app/screens/register/register.component.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../scss/colors';
+
+.register-form {
+  align-items: center;
+  background-color: $wild-sand;
+  border-top: 4px solid $blue;
+  display: flex;
+  flex-direction: column;
+  left: 50%;
+  max-width: 350px;
+  padding: 30px 20px 20px;
+  position: fixed;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 100%;
+
+  .logo {
+    max-height: 40px;
+  }
+
+  .title {
+    letter-spacing: 6px;
+    margin: 10px 0 30px;
+    text-transform: uppercase;
+  }
+
+  .label,
+  .input,
+  .btn,
+  .separator {
+    width: 100%;
```

Comment:
this should be automatic If you remove the align items-center in the form and just set an align-self in the image which is the only non full width item